libhoney/src/test/java/io/honeycomb/libhoney/utils/ObjectUtilsTest.java

```diff
@@ -12,15 +12,21 @@
 
     @Test
     public void checkThatTheFormatterCanHandleZuluTime() throws ParseException {
-        final Date parse = ObjectUtils.getRFC3339DateTimeFormatter().parse("2018-05-01T12:01:00.0925Z");
+        final Date parse = ObjectUtils.getRFC3339DateTimeFormatter().parse("2018-05-01T12:01:00.925Z");
 
+        assertThat(parse).hasMinute(01); 
+        assertThat(parse).hasSecond(00);
```

Comment:
using zero values makes me mildly anxious; could you set the test time to 23 or something instead?